views/about_us.ejs

```diff
@@ -225,7 +225,7 @@
 <div class="row demo-tiles">
     <div class="col-xs-12 col-sm-6 col-md-4 bio">
         <div class="tile">
-            <img src="/images/about-us/bio/2015-2016/noah-patullo.png" class="img-circle tile-image" alt="Noah Patullo">
+            <img src="/images/about-us/bio/2015-2016/noah-patullo.jpg" class="img-circle tile-image" alt="Noah Patullo">
```

Comment:
Thanks for catching it
Fixed now https://github.com/books4equality/books4equality/pull/85/commits/411ac4e58e34cc462c511901c603133122c39a3e